src/service/link-rewriter/link-rewriter.js

```diff
@@ -144,9 +193,10 @@ export default class LinkRewriter {
   }
 
   /**
-  * Update the state of the internal Anchor to replacement url Map.
-  * @param {*} anchorReplacementTupleList
-  */
+   * @private
+   * Update the state of the internal Anchor to replacement url Map.
```

Comment:
`Update the state of the internal (anchor => replacement url) map.`

Reply:
This is gone in an other branch